spec/lib/parameter_spec.rb

```diff
@@ -76,5 +76,22 @@ def param(identifier, type, default, nullable)
         end
       end
     end
+
+    describe "user_id type" do
+      fab!(:user)
+      it "raises an error if no such user exists" do
+        expect {
+          param("user_id", :user_id, nil, false).cast_to_ruby("user_not_exist")
+        }.to raise_error(::DiscourseDataExplorer::ValidationError)
+        expect {
+          param("user_id", :user_id, nil, false).cast_to_ruby("user_not_exist@fake.email")
+        }.to raise_error(::DiscourseDataExplorer::ValidationError)
```

Comment:
It will display the running results as before. Since user_id is a user selector, the possibility of entering this error is actually very small - this requires the user to enter an incorrect initial/default value and submit it.